engine/runtime/src/main/java/org/enso/interpreter/instrument/job/BackgroundJob.java

```diff
@@ -0,0 +1,23 @@
+package org.enso.interpreter.instrument.job;
+
+import scala.collection.immutable.List$;
+
+/** The job that runs in the background. */
+public abstract class BackgroundJob<A> extends Job<A> implements Comparable<BackgroundJob<?>> {
+
+  private final int priority;
+
+  public BackgroundJob(int priority) {
```

Comment:
How do we interpret priority? Lower the sooner?

Reply:
Yes, I'll add a comment